Dockerfile

```diff
@@ -466,7 +466,7 @@ COPY bin/fail /usr/bin/
 #===============
 # TODO: Use Google fingerprint to verify downloads
 #  https://www.google.de/linuxrepositories/
-ARG EXPECTED_CHROME_VERSION="89.0.4389.114"
+ARG EXPECTED_CHROME_VERSION="90.0.4430"
```

Comment:
Are You sure this is a valid version? Comparing to e.g. https://chromereleases.googleblog.com/2021/04/stable-channel-update-for-desktop_14.html?